docs/howto/kubernetes/gardener/create-shoot-cluster.md

```diff
@@ -0,0 +1,113 @@
+# Create Kubernetes cluster using Cleura Cloud Control Panel 
+<!-- Config interpolation for the gui name didn't work for the title here. -->
+
+If you want to create a Kubernetes cluster, you can do it via {{config.extra.gui}} using Gardener.
+This tutorial shows you how to do that, and how to deploy a sample application on such cluster.
+
+## Prerequisites
+
+To access the cluster from your computer, you will need [kubectl](https://kubernetes.io/docs/tasks/tools/) installed on your machine.
+
+## Creating Kubernetes cluster in {{config.extra.gui}}
+
+To get started, navigate to [{{config.extra.gui_domain}}](https://{{config.extra.gui_domain}}) and in the side panel choose Kubernetes
+-> [Managed Kubernetes](https://{{config.extra.gui_domain}}/containers/gardener).
+You will be presented with a Gardener page, in which you can create and manage your clusters. Click `Create Kubernetes cluster`.
+
+![Gardener page in {{config.extra.gui}}](assets/gardener_page.png)
+
+> In Gardener's terminology, kubernetes cluster is referred as a **shoot cluster**. You can see name "Shoot" in various places throughout
```

Comment:
"Kubernetes" is a proper name, so it should always use title case (i.e. a capital first letter). Also, "*a* Kubernetes cluster", and "*the* name 'Shoot'".